### PR TITLE
Feature/scenes

### PR DESCRIPTION
**Pull request by yurilov**
Bet panel components is now present on screen
Still need to continue work (postioning, additional methods, etc) ,  especially i had troubles with using assets from Loader.
Dont have time left today, so i ll continue to work with tommorow in the afternoon.

### CODE REVIEW

File: pixiv6/src/slotMachine/betPanel/balance/Balance.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { Container } from '@pixi/display';
+import { Graphics } from '@pixi/graphics';
+import BetPanel from '../BetPanel';
+import StyledText from 'app/slotMachine/styledText/StyledText';
+const margin = (1080 - 150 * 3) / 2
```
Comment (ValentynZubko) [inline pixiv6/src/slotMachine/betPanel/balance/Balance.ts:5]: 1. Defined constants should name with all uppercase letters and use "_" between words. For example "const MARGIN_TOP: number = (1080 - 150 * 3) / 2".
2. It's better to move all constants inside the class related to it

Comment (ValentynZubko) [inline pixiv6/src/slotMachine/betPanel/balance/Balance.ts:5]: 1. Defined constants should name with all uppercase letters and use "_" between words. For example "const MARGIN_TOP: number = (1080 - 150 * 3) / 2".
2. It's better to move all constants inside the class related to it

File: pixiv6/src/slotMachine/betPanel/betSelector/BetSelector.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import { Container } from "@pixi/display";
+import BetPanel from "../BetPanel";
+import StyledText from "app/slotMachine/styledText/StyledText";
+import { betList } from "./betList";
+const margin = (1080 - 150 * 3) / 2
```
Comment (ValentynZubko) [inline pixiv6/src/slotMachine/betPanel/betSelector/BetSelector.ts:5]: This constant is already defined in the Balance class, it's bad practice to duplicate constants. Choose one place for constant and make it static to have access to it from other classes

Comment (ValentynZubko) [inline pixiv6/src/slotMachine/betPanel/betSelector/BetSelector.ts:5]: This constant is already defined in the Balance class, it's bad practice to duplicate constants. Choose one place for constant and make it static to have access to it from other classes

File: pixiv6/src/slotMachine/betPanel/lines/Lines.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import { Container } from '@pixi/display';
+import StyledText from 'app/slotMachine/styledText/StyledText';
+import BetPanel from '../BetPanel';
+const margin = (1080 - 150 * 3) / 2
```
Comment (ValentynZubko) [inline pixiv6/src/slotMachine/betPanel/lines/Lines.ts:4]: Same constant

Comment (ValentynZubko) [inline pixiv6/src/slotMachine/betPanel/lines/Lines.ts:4]: Same constant

File: pixiv6/src/slotMachine/betPanel/playButton/PlayButton.ts [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import { Graphics } from '@pixi/graphics';
+import BetPanel from '../BetPanel';
+import StyledText from 'app/slotMachine/styledText/StyledText';
+const margin = (1080 - 150 * 3) / 2
```
Comment (ValentynZubko) [inline pixiv6/src/slotMachine/betPanel/playButton/PlayButton.ts:4]: Same constant

Comment (ValentynZubko) [inline pixiv6/src/slotMachine/betPanel/playButton/PlayButton.ts:4]: Same constant

**Review by ValentynZubko — APPROVED** [review]
I'll merge it, but please fix comments during future tasks

File: pixiv6/src/Main.ts [hunk 5]
```diff
@@ -21,6 +21,9 @@ import PixiLayoutPlugin from "app/layoutManager/PixiLayoutPlugin";
 
 document.body.style.margin = "0";
 document.body.style.overflow = "hidden";
+// const heading = document.createElement('h1');
```
[the remark this answers is not in the document]
Comment (yurilov) [reply pixiv6/src/Main.ts:24]: this was just me testing if everything works, so i just deleted it

**Pull request merged**